src/Components/App/App.js

```diff
@@ -11,7 +11,7 @@ class App extends Component {
     super()
     this.state = {
       movies: [],
-      singleMovie: {},
+      singleMovie: null,
```

Comment:
Tell me about the decision to change this from an empty object to null - I dont have strong opinions either way but I'd like to hear yours - great interview fodder

Reply:
This was us just trying to do some debugging, no real reason for that change.

Reply:
I actually ended up needing to change this BACK to null, due to a guard clause I had in my single movie page that was checking to see if singleMovie was empty. Saved some code in that conditional.